extension/scripts/features/travel-table/ttTravelTable.js

```diff
@@ -303,7 +303,7 @@
 					});
 				}
 
-				if (!isFlying() && !isAbroad())
+				if (!isFlying() && !isAbroad() && document.find(".info-msg-cont.red .msg")?.textContent.indexOf("currently in a race") === -1)

```

Comment:
Checking the text seems redundant to me. There are only 2 reasons why there would be an error box on the travel page:
- you can't fly due to a race
- you can't fly due to a dirty bomb

With both it should not execute this box.